test/RewardPool/RewardPool.test.ts

```diff
@@ -360,29 +360,62 @@ export function RunVestedDelegateClaimTests(): void {
       ).to.be.revertedWith("Ownable: caller is not the owner");
     });
 
-    it("should return when active position", async function () {
-      const { systemValidatorSet, validatorSet, rewardPool, vestManager, vestManagerOwner, delegatedValidator } =
-        await loadFixture(this.fixtures.weeklyVestedDelegationFixture);
+    it("should return when active position & withdraw", async function () {
+      const {
+        systemValidatorSet,
+        validatorSet,
+        rewardPool,
+        liquidToken,
+        vestManager,
+        vestManagerOwner,
+        delegatedValidator,
+      } = await loadFixture(this.fixtures.weeklyVestedDelegationFixture);
 
       // ensure is active position
       expect(await rewardPool.isActiveDelegatePosition(delegatedValidator.address, vestManager.address), "isActive").to
         .be.true;
 
-      // reward to be accumulated
+      await time.increase(WEEK * 2);
+
+      await liquidToken.connect(vestManagerOwner).approve(vestManager.address, this.minDelegation.div(2));
+      await vestManager.cutVestedDelegatePosition(delegatedValidator.address, this.minDelegation.div(2));
+
+      await time.increase(WEEK);
       await commitEpoch(
         systemValidatorSet,
         rewardPool,
         [this.signers.validators[0], this.signers.validators[1], delegatedValidator],
         this.epochSize
       );
-      // withdraw previous amounts
-      await vestManager.withdraw(vestManagerOwner.address);
 
+      await vestManager.withdraw(vestManagerOwner.address);
       expect(
         await rewardPool.getRawDelegatorReward(delegatedValidator.address, vestManager.address),
         "getRawDelegatorReward"
       ).to.be.gt(0);
+      expect(await validatorSet.withdrawable(vestManager.address), "withdrawable").to.be.eq(0);
+    });
+
+    it("should return when active position & claim", async function () {
```

Comment:
What is the purpose of this test?

Reply:
apply needed checks in previous test and removed this one